client/src/components/profile/profile.js

```diff
@@ -26,14 +27,30 @@ class Profile extends Component {
       user: {},
       isFetching: false,
       isShowAlert: false,
+      isOwnerRole: 2, // id that refers to Restaurant Owner role in DB
```

Comment:
You can create separate file with constants where initialize this general constant and use better name (**is**Something - means true or false)
As you use id in DB, it can be changed in different environments (e.g. in other database, like MySql, in production or testing environments)